### PR TITLE
Pick Postgres and complete CRUD

### PR DESCRIPTION
**Pull request by smrutherford**

### CODE REVIEW

File: client/components/ReviewInteractor.jsx [hunk 1]
```diff
@@ -2,16 +2,16 @@ import React from 'react';
...
-axios.defaults.baseURL = '/reviews-module';
+// axios.defaults.baseURL = '/reviews-module';
 
 class ReviewInteractor extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      votedHelpful: this.props.review.votedHelpful,
-      votedNotHelpful: this.props.review.votedNotHelpful,
+      votedHelpful: this.props.review.voted_helpful,
```
Comment (yogurtandjam) [inline client/components/ReviewInteractor.jsx:11]: i noticed you changed the naming schema - is there a reason for this?

File: client/components/index.jsx [hunk 2]
```diff
@@ -17,15 +17,14 @@ class App extends React.Component {
...
   }
   componentDidMount() {
-    console.log('productId: ', this.state.productId);
     this.getReviews();
   }
   getReviews() {
     axios.get(`http://localhost:3002/reviews/${this.state.productId}/`)
       .then((res) => {
-        console.log('res.data: ', res.data);
+        console.log(res.data.rows)
```
Comment (yogurtandjam) [inline client/components/index.jsx:25]: do you want to keep these console logs in here?

File: client/components/index.jsx [hunk 3]
```diff
@@ -37,7 +36,7 @@ class App extends React.Component {
       <BodyWrapper>
         {
           this.state.reviews.map(review => (
-            <ReviewRow review={review} key={review.reviewId} />))
+            <ReviewRow review={review} key={review.reviewid} />))
```
Comment (yogurtandjam) [inline client/components/index.jsx:39]: i noticed you converted the other variables from camel to snake case - you should try to keep a consistent naming schema

File: server/server.js [hunk 4]
```diff
@@ -1,44 +1,45 @@
...
   console.log('received put request with: ', reviewId, field, value);
   db.updateReview(reviewId, field, value, () => {
     res.status(204).end();
   });
 });
 
-//  ******** CREATE & DELETE IN CRUD *************************** 
-app.post('/reviews/:id', (req, res) => {
-  const newReview = generator.generateReviewsData(req.params.id);
+/*  ******** CREATE & DELETE IN CRUD ***************************  */
```
Comment (yogurtandjam) [inline server/server.js:36]: i would put all your crud methods together since our Create, Read, Update , Delete methods are considered one api

**Review by yogurtandjam — APPROVED** [review]

File: client/components/index.jsx (hunk 3 above)
Comment (smrutherford) [reply client/components/index.jsx:39]: postgres forces everything to be lowercase so I had to change the camel case to snake case or all lowercase letters. Good catch on inconsistency though. I will change this to camel case

File: client/components/index.jsx (hunk 2 above)
Comment (smrutherford) [reply client/components/index.jsx:25]: will delete

File: client/components/ReviewInteractor.jsx (hunk 1 above)
Comment (smrutherford) [reply client/components/ReviewInteractor.jsx:11]: because postgres forces everything to be lowercase

**Pull request merged**